mLRS/modules/stm32-usb-device/usbd_conf.h

```diff
@@ -26,8 +26,16 @@ extern "C" {
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
+
+#ifdef STM32F072xB
 #include "stm32f0xx.h"
 #include "stm32f0xx_hal.h"
+#endif
```

Comment:
could you pl change this to 

#elif defined STM32G431xx

just cosmetics, I know

Reply:
done. also removed whitespaces